wallets/metamask/src/prepareExtension.ts

```diff
@@ -1,14 +1,26 @@
+import path from 'node:path'
 import { downloadFile, ensureCacheDirExists, unzipArchive } from '@synthetixio/synpress-cache'
+import fs from 'fs-extra'
 
 export const DEFAULT_METAMASK_VERSION = '11.9.1'
 export const EXTENSION_DOWNLOAD_URL = `https://github.com/MetaMask/metamask-extension/releases/download/v${DEFAULT_METAMASK_VERSION}/metamask-chrome-${DEFAULT_METAMASK_VERSION}.zip`
 
-export async function prepareExtension() {
-  const cacheDirPath = ensureCacheDirExists()
+export async function prepareExtension(forceCache = true) {
+  let outputDir = ''
+  if (forceCache) {
+    outputDir = ensureCacheDirExists()
+  } else {
+    outputDir = path.resolve('./', 'downloads')
+    outputDir = process.platform === 'win32' ? `file:\\\\\\${outputDir}` : outputDir
```

Comment:
why making it complicated, instead of doing
```
    outputDir = process.platform === 'win32' ? `file:\\\\\\${outputDir}` : path.resolve('./', 'downloads')
```
?

Reply:
are you sure it will work?